packages/client/lib/MetadataClient/CredentialOfferMetadataClient.ts

```diff
@@ -0,0 +1,22 @@
+import { CredentialOfferWithBaseURL, EndpointMetadata, ICredentialIssuerMetadataParametersV1_11 } from '@sphereon/openid4vci-common';
+
+import { MetadataClient } from './MetadataClient';
+
+export class CredentialOfferMetadataClient extends MetadataClient {
+  /**
+   * Retrieve metadata using the credentialOfferWithBaseURL obtained from a previous step
+   *
+   * @param credentialOfferWithBaseURL
+   */
+  public static async getServerMetaData(credentialOfferWithBaseURL: CredentialOfferWithBaseURL): Promise<EndpointMetadata> {
```

Comment:
Is there a good reason to split this up already? Why not simply accept both versions? It only is a name change at this point.

Reply:
reverted.